src/app/components/search/SavedSearch.js

```diff
@@ -70,12 +71,45 @@ const SavedSearch = ({ routeParams }) => (
                       }
                     `}
                     deleteMessage={
-                      <FormattedMessage
-                        id="savedSearch.deleteMessage"
-                        defaultMessage="Are you sure? This list is shared among all users of {teamName}. After deleting it, no user will be able to access it."
-                        description="A message that appears when a user tries to delete a list, warning them that it will affect other users in their workspace."
-                        values={{ teamName: props.saved_search.team ? props.saved_search.team.name : '' }}
-                      />
+                      props.saved_search?.is_part_of_feeds ?
+                        <>
+                          <FormattedHTMLMessage
+                            tagName="p"
+                            id="savedSearch.deleteMessageWarning"
+                            defaultMessage="Are you sure? This is shared among all users of <strong>{teamName}</strong>. After deleting it, no user will be able to access it <br /><br />"
+                            description="A message that appears when a user tries to delete a list, warning them that it will affect other users in their workspace."
+                            values={{
+                              teamName: props.saved_search?.team ? props.saved_search.team.name : '',
+                            }}
+                          />
+                          <Alert
+                            variant="warning"
+                            title={
+                              <FormattedHTMLMessage
+                                id="saveFeed.deleteCustomListWarning"
+                                defaultMessage="<strong>Deleting list will result in no content for the following shared feed(s):</strong>"
+                                description="Warning displayed on edit feed page when no list is selected."
+                              />
+                            }
+                            content={
+                              <ul className="bulleted-list">
+                                {props.saved_search?.feeds?.edges.map(feed => (
+                                  <li key={feed.node.id}>{feed.node.name}</li>
+                                ))}
+                              </ul>
+                            }
+                          />
+                        </>
+                        :
+                        <FormattedHTMLMessage
+                          tagName="p"
+                          id="savedSearch.deleteMessage"
+                          defaultMessage="Are you sure? This is shared among all users of <strong>{teamName}</strong> After deleting it, no user will be able to access it."
```

Comment:
nice @danielevalverde ! thanks for those changes. Just add the period that @amoedoamorim mentioned and we are good to go!

![danceparty](https://github.com/meedan/check-web/assets/418001/07f91469-4714-45cb-b0a1-0ff03e7b702a)